tests/components/link/link_display.test.ts

```diff
@@ -85,6 +85,14 @@ describe("link display component", () => {
     expect(fixture.querySelector(".o-link-tool")).toBeFalsy();
   });
 
+  test("right-click on a linked cell should show 'Edit Link' instead of 'Insert Link' in the context menu", async () => {
+    setCellContent(model, "A1", "[label](url.com)");
+    await rightClickCell(model, "A1");
+    expect(
+      fixture.querySelector(".o-menu .o-menu-item[data-name='insert_link']")?.textContent
+    ).toBe("Edit link");
+  });
+
```

Comment:
Maybe we can also add a test to check that we have the "Insert link" as textContent for a non-link cell :)